docs/release-procedure.md

```diff
@@ -121,3 +121,13 @@ Fore more information please check [Helm documentation](https://helm.sh/docs/top
 - upload the packaged chart to the release in this repository
 - update the [index.yaml](https://github.com/apache/incubator-yunikorn-release/blob/gh-pages/index.yaml) file in the gh-pages branch with the new release
 
+## Update the website
+
+- Create a new version of the [YuniKorn website](https://github.com/apache/incubator-yunikorn-site/tree/master).
+- Update the [download page](http://yunikorn.apache.org/community/download) of the website.
+- Publish release announcement in the [apache Blog](https://blogs.apache.org/yunikorn/) and the the [YuniKorn website](http://yunikorn.apache.org).
```

Comment:
there is an extra "the"
and we should mention the release notes should be under versioned doc, like: http://yunikorn.apache.org/docs/get_started/release_notes

Reply:
Ah indeed. Added the extra mention as well.